core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -28,7 +31,7 @@
     </svg>
   </div>
   <!-- Left-to-right is enforced here due to the formatting of the HTML with the title on the left and chapter count on the right. -->
```

Comment:
Given your change below, is this comment still relevant?

Reply:
No, removed